### PR TITLE
data: Deepakktripathy

### PR DESCRIPTION
**Pull request by Deepakktripathy**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
You all are doing amazing! Would love to have my linkfree profile created!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4701"></a>

### CODE REVIEW

File: data/Deepakktripathy.json
Comment (priyankarpal) [inline data/Deepakktripathy.json:3]: Currently, you don't need to add the `"displayStatsPublic" : true,` code. you can delete it.
Check the updated doc https://linkfree.eddiehub.io/docs/quickstart

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by Deepakktripathy — COMMENTED** [review]
Removed displayStatsPublic

**Review by Hrishi5111998 — APPROVED** [review]

**Comment by Deepakktripathy** [comment]
Hey, I am unable to view my linkfree profile. It still shows "user not found"

**Comment by priyankarpal** [comment]
> Hey, I am unable to view my linkfree profile. It still shows "user not found"

Wait Maintainer will merge it soon & after merging it will automatically live.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**
